sideboard/server.py

```diff
@@ -93,7 +93,8 @@ def list_plugins(self):
                 'paths': []
             }
         for path, app in cherrypy.tree.apps.items():
-            if path:  # exclude what Sideboard itself mounts
+            # exclude what Sideboard itself mounts and grafted mount points
+            if path and hasattr(app, 'root'):
```

Comment:
This makes the ``list_plugins`` page (which MAGFest hides behind Nginx) work with Django apps which are grafted onto CherryPy.  (I'm using "grafted" as a technical term here since ``cherrypy.tree.graft`` is the actual API call in question.)